frontend/src/sheltersTest.ts

```diff
@@ -0,0 +1,43 @@
+export type Shelter = {
+  id: number;
+  title: string;
+  description: string;
+  latitude: number;
+  longitude: number;
+  emoji: string;
+};
+
+export const shelters: Shelter[] = [
```

Comment:
Not a concern for now, but I'll get the tables we need up in Dynamo so we can work on connecting frontend and backend in the spring